HotelApi/Startup.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Web.Http;
+using Microsoft.Owin;
+using Owin;
+
+[assembly: OwinStartup(typeof(HotelApi.Startup))]
+
+namespace HotelApi
+{
+    public class Startup
+    {
+        public void Configuration(IAppBuilder app)
+        {
+            HttpConfiguration config = new HttpConfiguration();
+            config.Routes.MapHttpRoute(
+                name: "DefaultApi",
+                routeTemplate: "api/{controller}/{action}",
```

Comment:
Couple of things here:
* usually in Apis we do not using action name we rather follow convention (controller method start with HTTP verb name)described [here](https://docs.microsoft.com/en-us/aspnet/web-api/overview/web-api-routing-and-actions/routing-in-aspnet-web-api)
* if you have something like info page it would be nice to add default controller and maybe action
* you do not need Id param, you have propably copied it from some external example - could you please get rid of this param.

Reply:
applied